test/lsf.jl

```diff
@@ -0,0 +1,14 @@
+if success(open(`which bsub`))
```

Comment:
Could `success(open(`which bsub`))` be abstracted as a function? We may want to declare an alternate version of this function on Windows for example.

Reply:
wow, i'm surprised an LSF client can be installed on windows!  so you want sth like:

```
lsf_installed() = Sys.isunix() && success(open(`which bsub`))
```

and then down the line someone with a windows box to test on can expand on that?

Reply:
```
julia> windows_which(command) = `powershell.exe -Command Get-Command $command`
windows_which (generic function with 1 method)

julia> success(windows_which("notepad.exe"))
true

julia> success(windows_which("asdf.exe"))
false
```

I was thinking about something like this:
```
@static if Sys.iswindows()
    windows_which(command) = `powershell.exe -Command Get-Command $command`
    is_lsf_installed() = success(windows_which(bsub))
else
   # do we actually need open?
    is_lsf_installed() = success(open(`which bsub`))
end
```

I'm not sure if you need to call `open` here.

Reply:
I have not used the cluster yet, but it is about time that I do.